zeebe/msgpack-value/src/main/java/io/camunda/zeebe/msgpack/property/BaseProperty.java

```diff
@@ -52,6 +52,10 @@ public boolean hasValue() {
     return isSet || defaultValue != null;
   }
 
+  public boolean isSet() {
+    return isSet;
+  }
```

Comment:
So `hasValue()` returns true if `isSet` or if there is a default. With this helper, in the follow up PR, I can check if the type was PIN, but there is actually no value set. But you're right, once we serialize, I guess this won't matter, so this is pointless :(

If that's OK with you, to avoid stack rebasing, I will remove it in the last PR?

Reply:
Or nah, I'll just do it here to avoid polluting the history. It's on me for stacking things :smile: